modules/garbage-collection.adoc

```diff
@@ -41,6 +41,9 @@ The following configuration fields are available to customize what is garbage co
 | **PUSH_TEMP_TAG_EXPIRATION_SEC** | The number of seconds that blobs will not be garbage collected after being uploaded. This feature prevents garbage collection from cleaning up blobs that are not referenced yet, but still used as part of an ongoing push. |String
 | **TAG_EXPIRATION_OPTIONS** | List of valid tag expiration values. |String
 | **DEFAULT_TAG_EXPIRATION** | Tag expiration time for time machine. |String
+| **CLEAN_BLOB_UPLOAD_FOLDER** | When set to `true`, automatically cleans stale blobs left over from an S3 multipart upload. | Boolean
++
+**Default:** `false`  
```

Comment:
It is better to highlight quay will clean up the blob files older then 2 days every a hour.  Customer care about these value very much. 

We can keep customer in the dark with BLOBUPLOAD_DELETION_DATE_THRESHOLD and BLOBUPLOAD_CLEANUP_FREQUENCY configurations.  but we need to let customer know the default behaviour. When sales describe this function to customer,  these data are necessary for sales. 

I remember the default value is
``` 
BLOBUPLOAD_DELETION_DATE_THRESHOLD   2 days 
BLOBUPLOAD_CLEANUP_FREQUENCY   one hour
```
Brandon can help to double confirm with these value.

Reply:
These are the correct values. It does seem good to include this information.